src/util/validationUtils.js

```diff
@@ -78,12 +78,22 @@ const validateCareGapsParams = query => {
       );
     }
   } else if (query.subject) {
+    if (query.practitioner) {
+      throw new BadRequestError('Must provide either subject or practitioner. Received both');
```

Comment:
I'd rephrase this error message. subject and practitioner are not direct replacements for each other. Maybe try something simple like "Cannot provide both a subject and practitioner"